test/suite/test_backup18.py

```diff
@@ -68,6 +75,10 @@ def test_backup18(self):
         # Open up the backup cursor.
         config = 'incremental=(enabled,this_id="ID1")'
         bkup_c = self.session.open_cursor('backup:', None, config)
+        backup = self.get_stat(stat.conn.backup_cursor_open)
+        self.assertEqual(backup, 1)
+        backup = self.get_stat(stat.conn.backup_incremental)
+        self.assertEqual(backup, 1)
```

Comment:
This is not so readable when all values are assigned to `backup`.  Maybe pick names like `backup_cursor_open = self.get_stat(....)`

Or could do:
```
    self.assertEqual(1, self.get_stat(stat.conn.backup_cursor_open))
```